priv/repo/migrations/20170717092127_allow_user_email_username_null.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CodeCorps.Repo.Migrations.AllowUserEmailUsernameNull do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      modify :email, :string, null: true
+      modify :username, :string, null: true
```

Comment:
I could technically leave this one out and set the `username` to be the same as `github_username`, but I feel it just confuses things as we later merge a new/newly-connected user with this one.